aspnetcore/security/authentication/accconfirm.md

```diff
@@ -98,53 +156,97 @@ This tutorial shows how to add email notification through [SendGrid](https://sen
+The template has the code for account confirmation and password recovery.  Find the `[HttpPost] Register` method in the  *AccountController.cs* file.
+
+# [ASP.NET Core 2.x](#tab/aspnet20)
+
+Prevent newly registered users from being automatically logged on by commenting out the following line:
+
+```csharp 
+await _signInManager.SignInAsync(user, isPersistent: false);
+```
+
+The complete method is show with the changed line highlighted:
+
+[!code-csharp[Main](accconfirm/sample/WebPW/Controllers/AccountController.cs?highlight=19&name=snippet_Register)]
+
+# [ASP.NET Core 1.x](#tab/aspnet1x)
 
-*  Find the `[HttpPost] Register` method in the  *AccountController.cs* file.
-*  Uncomment the code to enable account confirmation.
+Uncomment the code to enable account confirmation.
 
 [!code-csharp[Main](accconfirm/sample/WebApp1/Controllers/AccountController.cs?highlight=16-25&name=snippet_Register)]
 
 Note: We're also preventing a newly registered user from being automatically logged on by commenting out the following line:
 
-
 ```csharp 
 //await _signInManager.SignInAsync(user, isPersistent: false);
 ```
 
-*  Enable password recovery by uncommenting the code in the `ForgotPassword` action in the *Controllers/AccountController.cs* file.
+Enable password recovery by uncommenting the code in the `ForgotPassword` action in the *Controllers/AccountController.cs* file.
 
 [!code-csharp[Main](accconfirm/sample/WebApp1/Controllers/AccountController.cs?highlight=17-23&name=snippet_ForgotPassword)]
 
 Uncomment the form element in *Views/Account/ForgotPassword.cshtml*. You might want to remove the `<p> For more information on how to enable reset password ... </p>` element which contains a link to this article.
 
 [!code-html[Main](accconfirm/sample/WebApp1/Views/Account/ForgotPassword.cshtml?highlight=7-10,12,28)]
 
+---
+
 ## Register, confirm email, and reset password
 
-In this section, run the web app and show the account confirmation and password recovery flow.
+Run the web app and test the account confirmation and password recovery flow.
 
 * Run the app and register a new user
 
  ![Web application Account Register view](accconfirm/_static/loginaccconfirm1.png)
 
-* Check your email for the account confirmation link. If you don't get the email notification:
-
-  * Check the SendGrid web site to verify your sent mail messages.
-  * Check your spam folder.
-  * Try another email alias on a different email provider (Microsoft, Yahoo, Gmail, etc.)
-  * In SSOX, navigate to **dbo.AspNetUsers** and delete the email entry and try again.
-
+* Check your email for the account confirmation link. See [Debug email](#debug) you don't get the email.
 * Click the link to confirm your email.
 * Log in with your email and password.
 * Log off.
 
+### View the manage page
+
+Select your user name in the browser:
+![browser window with username](accconfirm/_static/un.png)
+
+You might need to expand the navbar to see username.
```

Comment:
"user name" vs. "username"...let's standardize on one usage here. "user name" is used above.